docs/userguide/scheduling/propagate-dependencies.md

```diff
@@ -121,3 +129,324 @@ $  kubectl get configmap
+      replicaDivisionPreference: Weighted
+      replicaSchedulingType: Divided
+      weightPreference:
+        staticWeightList:
+          - targetCluster:
+              clusterNames:
+                - member1
+            weight: 1
+          - targetCluster:
+              clusterNames:
+                - member2
+            weight: 1
+---
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: app-b-policy
+  namespace: default
+spec:
+  conflictResolution: Abort
+  preserveResourcesOnDeletion: false
+  propagateDeps: true
+  resourceSelectors:
+    - apiVersion: apps/v1
+      kind: Deployment
+      name: app-b
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+    replicaScheduling:
+      replicaDivisionPreference: Weighted
+      replicaSchedulingType: Divided
+      weightPreference:
+        staticWeightList:
+          - targetCluster:
+              clusterNames:
+                - member1
+            weight: 1
+          - targetCluster:
+              clusterNames:
+                - member2
+            weight: 1
+```
+
+After applying these resources, Karmada will:
+
+1. Create independent ResourceBindings for `app-a` and `app-b` with their respective PropagationPolicy settings
+2. Create an attached ResourceBinding for the shared ConfigMap `shared-config`
+3. Detect the conflict between the two PropagationPolicy objects
+4. Resolve the conflict using the rules: `conflictResolution: Overwrite` (Overwrite wins), `preserveResourcesOnDeletion: true` (true wins)
+5. Emit a Warning event
+
+### Viewing Conflicts
+
+Check the attached ResourceBinding for the shared ConfigMap:
+
+```bash
+$ kubectl get resourcebindings
+$ kubectl get resourcebinding shared-config-configmap -o yaml
+```
+
+You will see the resolved values in the `spec`:
+
+```yaml
+apiVersion: work.karmada.io/v1alpha2
+kind: ResourceBinding
+metadata:
+  name: shared-config-configmap
+  namespace: default
+spec:
+  conflictResolution: Overwrite
+  preserveResourcesOnDeletion: true
+  resource:
+    apiVersion: v1
+    kind: ConfigMap
+    name: shared-config
+    namespace: default
+  requiredBy:
+    - name: app-a-deployment
+      namespace: default
+    - name: app-b-deployment
+      namespace: default
+```
+
+Check for conflict events:
+
+```bash
+$ kubectl get events -n default --field-selector reason=DependencyPolicyConflict
+```
+
+You will see a Warning event like:
+
+```
+LAST SEEN   TYPE      REASON                     OBJECT                                    MESSAGE
+2m42s       Warning   DependencyPolicyConflict   resourcebinding/shared-config-configmap   Dependency policy conflict detected: ConflictResolution conflicted (Overwrite vs Abort); PreserveResourcesOnDeletion conflicted (true vs false).
+```
+
+### Resolving Conflicts
+
+To eliminate the conflict warning, align the PropagationPolicy settings:
+
+**Option 1**: Update one PropagationPolicy to match the other
```

Comment:
The meaning here is to adjust the conflicting PropagationPolicy to be consistent. It doesn't necessarily refer to just one; there could be multiple conflicting PropagationPolicies.